### PR TITLE
Mixed logit page

### PR DESCRIPTION
**Pull request by jbazzle**
I added a page for mixed logit models that includes an example for R.  I also removed mixed logits from the desired non-existent pages md.

### CODE REVIEW

**Comment by NickCH-K** [comment]
Great, thank you! I'll merge as soon as the check completes.
